src/xen/public/arch-x86/cpuid.h

```diff
@@ -0,0 +1,118 @@
+/******************************************************************************
```

Comment:
fyi -- I did not look through all these headers. I assume it's all copy/paste

Reply:
Yes, they are imported (not all headers - only those required) verbatim from the xen tree (public headers only) with no changes.